browser/src/slideshow/transition/helper/PerformFadeTransition.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright the Collabora Online contributors.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+declare var SlideShow: any;
+
+enum FadeSubType {
+	FADEOVERBLACK,
+	FADEOVERWHITE,
+	SMOOTHLY,
+}
+
+function PerformFadeTransition(
```

Comment:
This class/function is not needed. This code can be included inside `FadeTransition`.
But if any type can use 2 different types depending on some properties -> then we should create such intermediate object